traffic_portal/app/src/modules/private/cdns/config/ConfigController.js

```diff
@@ -155,6 +155,14 @@ let ConfigController = function (cdn, currentSnapshot, newSnapshot, $scope, $sta
 		});
 	};
 
+	$scope.tabSelected = function() {
+		// issue 3863 - adjust column headers when tab is selected and data table is visible. hacky...sorry...
+		$timeout(function () {
```

Comment:
using `$timeout` triggers an `$apply` and a `$digest` cycle. Since you're not changing angular view data and are directly manipulating the DOM with JQuery libraries, there's no need to do that. So it'd be more performant to use [`window.setTimeout`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout) instead - should have exactly the same syntax.